src/containers/Cluster/Cluster.tsx

```diff
@@ -101,67 +102,127 @@ function Cluster({
     }, [nodes, versionToColor]);
 
-    const renderTab = () => {
-        switch (activeTab) {
-            case clusterTabsIds.tenants: {
-                return <Tenants additionalTenantsProps={additionalTenantsProps} />;
-            }
-            case clusterTabsIds.nodes: {
-                return (
-                    <NodesWrapper
-                        parentContainer={container.current}
-                        additionalNodesProps={additionalNodesProps}
-                    />
-                );
-            }
-            case clusterTabsIds.storage: {
-                return (
-                    <StorageWrapper
-                        parentContainer={container.current}
-                        additionalNodesProps={additionalNodesProps}
-                    />
-                );
-            }
-            case clusterTabsIds.versions: {
-                return <Versions versionToColor={versionToColor} />;
-            }
-            default: {
-                return null;
-            }
+    const getClusterTitle = () => {
+        if (infoLoading) {
+            return <Skeleton className={b('title-skeleton')} />;
         }
+
+        return (
+            <EntityStatus
+                size="m"
+                status={cluster?.Overall}
+                name={cluster?.Name ?? CLUSTER_DEFAULT_TITLE}
+                className={b('title')}
+            />
+        );
     };
 
     return (
         <div className={b()} ref={container}>
-            <ClusterInfo
-                cluster={cluster}
-                groupsStats={groupsStats}
-                versionsValues={versionsValues}
-                loading={infoLoading}
-                error={clusterError}
-                additionalClusterProps={additionalClusterProps}
-            />
-
+            <div className={b('header')}>{getClusterTitle()}</div>
             <div className={b('tabs')}>
                 <Tabs
                     size="l"
                     allowNotSelected={true}
-                    activeTab={activeTab as string}
+                    activeTab={activeTab}
                     items={clusterTabs}
                     wrapTo={({id}, node) => {
                         const path = getClusterPath(id as ClusterTab, queryParams);
                         return (
-                            <InternalLink to={path} key={id}>
+                            <InternalLink
+                                to={path}
+                                key={id}
+                                onClick={() => {
+                                    dispatch(updateDefaultClusterTab(id));
+                                }}
+                            >
                                 {node}
                             </InternalLink>
                         );
                     }}
                 />
             </div>
 
-            <div>{renderTab()}</div>
+            <div>
+                <Switch>
```

Comment:
What is the reason to use Router instead of switch case?

Reply:
It's easier here to redirect from `/cluster` to `/cluster/{defaultTab}` than to create proper references to the default tab throughout the codebase.